main/control/user.py

```diff
@@ -290,11 +296,12 @@ class UserMergeForm(wtf.Form):
 @app.route('/admin/user/merge/', methods=['GET', 'POST'])
 @auth.admin_required
 def user_merge():
-  user_keys = util.param('user_keys', list)
-  if not user_keys:
-    flask.abort(400)
+  args = parser.parse({
+      'user_key': wf.Str(missing=None),
```

Comment:
Not in all cases, [for example](https://gae-init.appspot.com/admin/user/merge/?user_keys=agpzfmdhZS1pbml0chELEgRVc2VyGICAgJDpwbQKDA,agpzfmdhZS1pbml0chELEgRVc2VyGICAgICAlr0KDA)